doc/customization_validation.rst

```diff
@@ -0,0 +1,9 @@
+.. SPDX-FileCopyrightText:  PyPSA-Earth and PyPSA-Eur Authors
+..
+.. SPDX-License-Identifier: CC-BY-4.0
+
+.. _customization_validation:
+
+################
+Model Validation
+################

```

Comment:
This section is empty. It may be good to move the validation in steps here

Reply:
Thanks for the comment. I have moved the validation part into this page.